### PR TITLE
Clarify github issues vs forum posts in docs.

### PR DESCRIPTION
**Pull request by ekpyron**
Maybe this helps a bit to guide the inflow of github issues.

### CODE REVIEW

File: docs/contributing.rst [hunk 1]
```diff
@@ -50,6 +50,10 @@ reporting issues, please mention the following details:
 Reducing the source code that caused the issue to a bare minimum is always
 very helpful, and sometimes even clarifies a misunderstanding.
 
+In case you want to start a language design discussion, a post in the
+`Solidity forum <https://forum.soliditylang.org/>`_ is preferable to
+a github issue (see :ref:`solidity_language_design`).
+
```
Comment (cameel) [inline docs/contributing.rst]: Just wanted to note that @NunoFilipeSantos recently suggested that this whole section should just be replaced with a link to the bug tracker and the text should simply be a part of the "new issue" template.

And in the template chooser we do have an option encouraging people to open forum discussions (added by @nikola-matic recently):
solidity-issue-template-chooser-forum-discussion

Comment (cameel) [reply docs/contributing.rst]: On the other hand, even if we just include a link, having this warning in multiple places might still be a good idea to draw people's attention. We could even make it a warning here.

**Review by cameel — COMMENTED** [review]
I'm fine with the change but not approving yet since I want @NunoFilipeSantos to take a look first.

File: docs/contributing.rst (hunk 1 above)
Comment (ekpyron) [reply docs/contributing.rst]: Ah, nice, that all sounds great - I'm also fine with dropping this in favour of those then.

File: docs/contributing.rst [hunk 2]
```diff
@@ -50,6 +50,10 @@ reporting issues, please mention the following details:
 Reducing the source code that caused the issue to a bare minimum is always
 very helpful, and sometimes even clarifies a misunderstanding.
 
+In case you want to start a language design discussion, a post in the
+`Solidity forum <https://forum.soliditylang.org/>`_ is preferable to
+a github issue (see :ref:`solidity_language_design`).
```
Comment (NunoFilipeSantos) [inline docs/contributing.rst]:
```suggestion
For technical discussions about language design, a post in the
`Solidity forum <https://forum.soliditylang.org/>`_ is the correct place (see :ref:`solidity_language_design`).
```

**Review by NunoFilipeSantos — CHANGES_REQUESTED** [review]
We should avoid the extra jump to GH.
If people still go there there's also a redirect in place.
The action in this doc should just be the correct link.

**Review by NunoFilipeSantos — APPROVED** [review]
LGTM.

**Review by cameel — APPROVED** [review]

**Comment by cameel** [comment]
CircleCI seems to be having some issues on macOS. I restarted the jobs but it's been stuck at "Preparing Environment" for 15 min now...

**Comment by cameel** [comment]
Since this is just a simple docs change and docs check did pass, I'll just force merge it.

**Pull request merged**